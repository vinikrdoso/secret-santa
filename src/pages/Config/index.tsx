import Card from '../../componentes/Card';
import Footer from '../../componentes/Footer';
import NameForm from '../../componentes/NameForm';
import ParticipantsList from '../../componentes/ParticipantsList';

import './styles.css';

const Config = () => {
	return (
		<Card>
			<section>
				<h2>Let's begin!</h2>
				<NameForm />
				<ParticipantsList />
				<Footer />
			</section>
		</Card>
	);
};

export default Config;
