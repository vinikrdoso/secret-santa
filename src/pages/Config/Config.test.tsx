import { render } from '@testing-library/react';
import { RecoilRoot } from 'recoil';
import Config from './index';

const mockNavigation = jest.fn();
jest.mock('react-router-dom', () => {
	return {
		useNavigate: () => mockNavigation,
	};
});

describe('Config page', () => {
	test('should render', () => {
		const { container } = render(
			<RecoilRoot>
				<Config />
			</RecoilRoot>
		);

		expect(container).toMatchSnapshot();
	});
});
