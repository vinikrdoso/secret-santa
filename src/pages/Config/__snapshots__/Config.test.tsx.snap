// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Config page should render 1`] = `
<div>
  <div
    class="card"
  >
    <section>
      <h2>
        Let's begin!
      </h2>
      <form>
        <div
          class="grupo-input-btn"
        >
          <input
            placeholder="Insert the names of all participants"
            type="text"
            value=""
          />
          <button
            disabled=""
          >
            Add
          </button>
        </div>
        
      </form>
      <ul />
      <footer
        class="footer-configs"
      >
        <button
          class="button"
          disabled=""
        >
          Start game
        </button>
        <img
          alt="Sacolas de compras"
          src="/images/sacolas.png"
        />
      </footer>
    </section>
  </div>
</div>
`;
